Firestore/core/src/firebase/firestore/util/firebase_assert.h

```diff
@@ -75,6 +75,10 @@
     }                                                                       \
   } while (0)
 
+// Assert with custom message that is not compiled out in release builds.
+#define FIREBASE_ASSERT_MESSAGE(expression, ...) \
```

Comment:
I took the liberty of defining a shorter name. If you have any suggestions on how to make it even shorter, I'll be happy to implement them, I find this SHOUTING pretty distractive.

Reply:
This change is in one of the other PRs too.

Unfortunately, these SHOUTING names that are already in the Firebase C++ SDK so we're stuck with them.

Reply:
> This change is in one of the other PRs too.

Can you link that PR? Did you mean to imply that I should roll this back?

Reply:
It's here https://github.com/firebase/firebase-ios-sdk/pull/733/files#diff-398e46f358d36158bce3e568e0a6ae48

You don't need to roll back, just be careful to merge correctly (and line up on the names now).